airflow/contrib/operators/gcs_to_gcs.py

```diff
@@ -143,51 +143,50 @@ def execute(self, context):
             google_cloud_storage_conn_id=self.google_cloud_storage_conn_id,
             delegate_to=self.delegate_to
         )
-        log_message = 'Executing copy of gs://{0}/{1} to gs://{2}/{3}'
+
+        if self.destination_bucket is None:
```

Comment:
Please add description of this behaviour to the docstring.

Reply:
Added